app/scripts/views/settings.js

```diff
@@ -148,6 +149,34 @@ function ($, modal, Cocktail, Session, BaseView, AvatarMixin,
       return self._showAvatar();
     },
 
+    beforeDestroy: function () {
+      $('body.settings').fadeOut(this.FADE_OUT_SETTINGS_MS, function (){
+        $('body').removeClass('settings').show();
+      });
+    },
+
+    // When the user adds, removes or changes a display name
+    // this gets called and swaps out headers to reflect
+    // the updated state of the account
+    _swapDisplayName: function () {
+      var account = this.getSignedInAccount();
+      var displayName = account.get('displayName');
+      var email = account.get('email');
+
+      var cardHeader = this.$('.card-header');
+      var cardSubheader = this.$('.card-subheader');
+
+      if (displayName === cardHeader.text()) {
+        return;
+      } else if (! displayName) {
+        cardHeader.text(email);
+        cardSubheader.text('');
+      } else {
+        cardHeader.text(displayName);
```

Comment:
This line and the next are not covered by unit tests:

![screen shot 2015-10-01 at 11 45 29](https://cloud.githubusercontent.com/assets/848085/10218852/0158e4d6-6832-11e5-8b3a-11b75e30523c.png)
.